packages/flutter_tools/test/general.shard/plugins_test.dart

```diff
@@ -3,10 +3,12 @@
 // found in the LICENSE file.
 
 import 'dart:convert';
+import 'dart:io';
```

Comment:
nit. do you *really* need to import `dart:io` here, or would `package:flutter_tools/src/base/io.dart` be sufficient?

Reply:
Tried to fix this locally, but it appears io.dart doesn't expose PathNotFoundException?

Reply:
Tried to fix this locally, but it appears io.dart doesn't expose PathNotFoundException?

Reply:
Ahh, ok, this LGTM then